mParticle-Apptimize/MPKitApptimize.m

```diff
@@ -191,8 +191,11 @@ - (nonnull MPKitExecStatus *)setUserIdentity:(NSString *)identityString identity
 
 #pragma mark Events
 
-- (nonnull MPKitExecStatus *)logEvent:(MPEvent *)event {
-    [Apptimize track:event.name];
+- (nonnull MPKitExecStatus *)logBaseEvent:(nonnull MPBaseEvent *)event {
+    if (![event isKindOfClass:[MPEvent class]]) {
```

Comment:
Hi--thanks for this PR. Can you please update this method to return unavailable status if the message can not be handled? This allows us to create accurate reporting stats for how many messages were forwarded.

Also it looks like your if statement may be inverted, here's one possible adjustment to how the code might look:

```
    if ([event isKindOfClass:[MPEvent class]]) {
        MPEvent *mpEvent = (MPEvent*)event;
        [Apptimize track:mpEvent.name];
	return [self makeStatus:MPKitReturnCodeSuccess];
    }
    return [self makeStatus:MPKitReturnCodeUnavailable];
```

Finally, it's probably best to avoid setting the version number in the PR to avoid merge conflicts--we typically bump at time of release via automation.